src/pages/Home.js

```diff
@@ -1,9 +1,13 @@
 
 import styled from "styled-components";
 import { useNavigate } from "react-router-dom";
+import { useState } from "react";
 
 export default function Home() {
   const navigate = useNavigate();
+  const [buttonBox1, setButtonBox1] = useState(false);
+  const [buttonBox2, setButtonBox2] = useState(false);
+  const [buttonBox3, setButtonBox3] = useState(false);
   const user = localStorage.getItem(`user`);
 
   if(user === null) {
```

Comment:
unnecessary fragments in line 14 & 16. Please remove

Reply:
removed